packages/pasteup/palette.ts

```diff
@@ -116,7 +123,12 @@ const labs = {
     main: '#69d1ca',
 };
 
-export const palette: AllPillarColours & OtherColours = {
+const contrasts = {
+    darkOnLight: { background: neutral[97], foreground: neutral[20] },
```

Comment:
```suggestion
    darkOnLight: { background: neutral[100], foreground: neutral[20] },
```

should this be pure white?

Reply:
This is what was defined before, do you have an example of the component used on white?

Reply:
no idea about this specific button but I'd figure if it were to be used on white it should look like our text-based grey hollow buttons?

![Screenshot 2019-04-10 at 15 58 14](https://user-images.githubusercontent.com/11539094/55889788-8924c600-5ba9-11e9-94aa-4bbe6eca8a4e.png)

@zeftilldeath help us out here

https://support-ui.gutools.co.uk/?selectedKind=Buttons&selectedStory=Button%20button&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs